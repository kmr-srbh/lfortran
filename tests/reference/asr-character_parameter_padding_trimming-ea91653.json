{
    "basename": "asr-character_parameter_padding_trimming-ea91653",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/character_parameter_padding_trimming.f90",
    "infile_hash": "3ac17fa68feb60423eaa66ebbb88a3e18da1a72d0c7e8ed8ac8de022",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-character_parameter_padding_trimming-ea91653.stderr",
    "stderr_hash": "7435da5a3769b1b237c9999f77f6e8766604928d96cda18974248e80",
    "returncode": 2
}