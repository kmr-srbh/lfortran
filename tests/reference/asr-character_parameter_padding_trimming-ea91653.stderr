semantic error: The RHS character len must be known at compile time
 --> tests/character_parameter_padding_trimming.f90:6:5
  |
6 |     character(len=8), parameter :: x_pad = "apple" !> char length on right is 5
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 
